src/main/java/com/notably/commons/core/path/AbsolutePath.java

```diff
@@ -33,7 +33,7 @@ public AbsolutePath(List<String> absolutePathList) {
      * @return
      */
     public static boolean isValidAbsolutePath(String absolutePathString) {
-        return absolutePathString.matches(VALIDATION_REGEX);
+        return absolutePathString.matches(VALIDATION_REGEX) || absolutePathString.equals("/");
```

Comment:
Hey @ljiazh3ng, This should be done on the `VALIDATION_REGEX` level to maintain consistency!

You can modify `VALIDATION_REGEX` from `(\\/\\p{Alnum}+)+\\/?` to `\\/|(\\/\\p{Alnum}+)+\\/?` to achieve this